RadixWallet/Features/TransactionReviewFeature/TransactionReviewDepositExceptions/TransactionReviewDepositExceptions.swift

```diff
@@ -0,0 +1,38 @@
+import ComposableArchitecture
+import SwiftUI
+
+// MARK: - TransactionReviewDepositExceptions
+public struct TransactionReviewDepositExceptions: Sendable, FeatureReducer {
```

Comment:
Hmm, do we really need a TCA stack - Reducer + View? Can't we have TransactionReviewDepositExceptionsView which accepts a ViewState?